src/Dispatcher/Masa.Contrib.Dispatcher.IntegrationEvents.Dapr/Processor/InfiniteLoopProcessor.cs

```diff
@@ -20,15 +20,16 @@ public override async Task ExecuteAsync(CancellationToken stoppingToken)
                 await _processor.ExecuteAsync(stoppingToken);
                 await DelayAsync(((ProcessorBase)_processor).Delay);
             }
-            catch (OperationCanceledException)
+            catch (OperationCanceledException ex)
             {
                 //ignore
+                _logger?.LogWarning("Operation canceled", ex);
             }
             catch (Exception ex)
             {
                 _logger?.LogWarning(ex, "Processor '{ProcessorName}' failed", _processor.ToString());
 
-                Thread.Sleep(TimeSpan.FromSeconds(2));
+                await DelayAsync(2);
```

Comment:
Why delay 2 seconds?

Reply:
Release the current task and execute other tasks to avoid an infinite loop，There is no strict constraint on whether to release 2 seconds or 5 seconds